pom.xml

```diff
@@ -632,12 +632,20 @@
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>1.4.5</version>
+                        <version>3.0.2</version>
                         <configuration>
                             <failBuildOnCVSS>7</failBuildOnCVSS>
                             <format>ALL</format>
-                            <suppressionFile>dependency-check-maven-config.xml</suppressionFile>
+                            <suppressionFiles>
+                                <suppressionFile>https://raw.githubusercontent.com/codice/ddf/master/dependency-check-maven-config.xml</suppressionFile>
+                                <suppressionFile>https://raw.githubusercontent.com/codice/ddf-support/master/support-owasp/src/main/resources/dependency-check-maven-config.xml</suppressionFile>
```

Comment:
Do we want to include this suppression file? I don't think it is maintained anymore.

Reply:
I wasn't sure if it was being maintained or not. I can remove it.